pyepsg_test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import pyepsg
+import datetime
+
+class TestAll(unittest.TestCase):
+
+    def test_get_cached(self):
+        #on first pass, cache wont be live, so it will take longer (will fetch from URL)
+        start_time = datetime.datetime.now()
+        proj1 = pyepsg.get(21781)
+        e = datetime.datetime.now() - start_time
+        self.assertGreater(e.total_seconds(), 0.01)
+        self.assertTrue('PROJCS["CH1903_LV03",GEOGCS["GCS_CH1903",DATUM["D_CH1903",SPHEROID["Bessel_1841",6377397.155,299.1528128]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]],PROJECTION["Hotine_Oblique_Mercator_Azimuth_Center"],PARAMETER["latitude_of_center",46.95240555555556],PARAMETER["longitude_of_center",7.439583333333333],PARAMETER["azimuth",90],PARAMETER["scale_factor",1],PARAMETER["false_easting",600000],PARAMETER["false_northing",200000],UNIT["Meter",1]]', proj1.as_esri_wkt())
+
+        #now it must be immediate as cache was already populated
+        start_time = datetime.datetime.now()
+        proj2 = pyepsg.get(21781)
+        e = datetime.datetime.now() - start_time
+        self.assertLess(e.total_seconds(), 0.01)
+        self.assertTrue('PROJCS["CH1903_LV03",GEOGCS["GCS_CH1903",DATUM["D_CH1903",SPHEROID["Bessel_1841",6377397.155,299.1528128]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]],PROJECTION["Hotine_Oblique_Mercator_Azimuth_Center"],PARAMETER["latitude_of_center",46.95240555555556],PARAMETER["longitude_of_center",7.439583333333333],PARAMETER["azimuth",90],PARAMETER["scale_factor",1],PARAMETER["false_easting",600000],PARAMETER["false_northing",200000],UNIT["Meter",1]]', proj2.as_esri_wkt())
```

Comment:
Using the execution time to detect caching is not safe. For example, on a machine that accesses the internet through a caching web proxy even the first call to `pyepsg.get` may take less than 0.01 s. There are `cache_info()` and `cache_clear()` functions provided by the decorator to avoid the need for inferring behaviour in this way. But...

Effectively this is just testing the behaviour of functools.lru_cache which is not our responsibility.

So, thank you for adding a test, but I don't think the addition of the lru_cache decorator really warrants one. I suggest we remove this file for now.